src/mesh.c

```
@@ -6,381 +6,315 @@
     mesh_send_beacon();
 
     return 0;
 }
 
-/**
- * Set application callback for received packets
- */
-int mesh_set_rx_callback(void (*callback)(const packet_t *packet, int16_t rssi, int8_t snr))
-{
-    mesh_state.rx_callback = callback;
+int mesh_set_rx_callback(void (*callback)(const packet_t* packet, int16_t rssi, int8_t snr)) {
```

Cppcheck (2.21.0)

[style] The function 'mesh_set_rx_callback' is never used.

(unusedFunction)